src/store/modules/map/index.js

```diff
@@ -31,15 +35,15 @@ export default {
     },
     UPDATE_RASTER_LAYER_OPACITY(state, { id, opacity }) {
       const layerToUpdate = state.rasterLayers.find(layer => layer.id === id)
+      const index = state.rasterLayers.findIndex(layer => layer.id === id)
 
-      if (!layerToUpdate) { return }
+      if (index === -1) {
+        return
+      }
```

Comment:
I found the previous method more readable. `layerToUpdate` is still `undefined`, now I have to know what `index` is, and that an index of `-1` means, **non existent**.

The previous version (`if (!layerToUpdate) return`), reads more like english: if no layerToUpdate, return

Reply:
Reverted.